src/views/AdminSettings/GeneralSettings.vue

```diff
@@ -112,6 +133,15 @@ export default {
 				}.bind(this),
 			})
 		},
+		saveDefaultGroupNotification() {
+			this.loadingStartCalls = true
+
+			OCP.AppConfig.setValue('spreed', 'default_group_notification', this.startCalls.value, {
```

Comment:
```suggestion
			OCP.AppConfig.setValue('spreed', 'default_group_notification', this.defaultGroupNotification.value, {
```

Reply:
Thanks, changed it. 
But nevertheless, all group participants have no chat notification set, it's still empty:
![grafik](https://user-images.githubusercontent.com/24530680/73297048-1b668b00-420b-11ea-97b4-7300330169c4.png)